fli-iam/shanoir-ng
# [Fli iam#2082] Add fields to the OFSEP stats export

# Summary
- add 2 fields (center_postcode, center_city) to the stats export
- on OFSEP sequence identifier post-processing, save two new datset properties extracted from dataset DICOM ("dicom.InstitutionName", "dicom.InstitutionAddress").

# How to test
- deploy on OSEP qualif
- execute VIP `ofsep_sequences_identification/0.2` pipeline on at least 1 dataset
- download statistics (may take several minutes)
- check that new column "center_postcode" and "center_city" are present and well filled
- check that, for the relevant datasets, new JSON properties "dicom.InstitutionName" and "dicom.InstitutionAddress" are present in the execution column and well filled
